libsolidity/CompilerStack.cpp

```diff
@@ -179,6 +192,24 @@ string const& CompilerStack::getMetadata(string const& _contractName, Documentat
 	return *(*doc);
 }
 
+std::string const CompilerStack::getFunctionHashes(std::string const& _contractName)
```

Comment:
This looks like pretty-printing for the user interface to me and I think it should not belong into this class.